SPEC.md

```diff
@@ -51,10 +51,13 @@ B. Container Runtime
 - The device configuration includes mandatory fields such as "name" and "version".
 - Fields in CDI structures are required unless specifically marked optional.
 
-For the purposes of this proposal, we define two terms very specifically:
+For the purposes of this proposal, we define the following terms:
 - _container_ can be considered synonymous with a [Linux _network namespace_][namespaces].
-  What unit this corresponds to depends on a particular container runtime implementation. 
-- _device_ which will refer to an actual hardware device.
+  What unit this corresponds to depends on a particular container runtime implementation.
+- _device_ which refers to an actual hardware device.
+- _runtime engine_ which refers to the component that ingests the OCI specification and creates a container, e.g: runc, crun, ...
+- _container runtime_ which refers to the higher level component users tend to interact with, e.g: docker, podman, ...
```

Comment:
interact with for managing containers... may also include lower level components that implement management of containers and pods (sets of containers).

Reply:
Addressed